packages/hoppscotch-sh-admin/src/helpers/auth.ts

```diff
@@ -78,142 +73,87 @@ const signOut = async (reloadWindow = false) => {
   });
 };
 
-async function signInUserWithGithubFB() {
+const signInUserWithGithubFB = async () => {
   window.location.href = `${
     import.meta.env.VITE_BACKEND_API_URL
   }/auth/github?redirect_uri=${import.meta.env.VITE_ADMIN_URL}`;
-}
+};
 
-async function signInUserWithGoogleFB() {
+const signInUserWithGoogleFB = async () => {
   window.location.href = `${
     import.meta.env.VITE_BACKEND_API_URL
   }/auth/google?redirect_uri=${import.meta.env.VITE_ADMIN_URL}`;
-}
+};
 
-async function signInUserWithMicrosoftFB() {
+const signInUserWithMicrosoftFB = async () => {
   window.location.href = `${
     import.meta.env.VITE_BACKEND_API_URL
   }/auth/microsoft?redirect_uri=${import.meta.env.VITE_ADMIN_URL}`;
-}
+};
 
-async function getInitialUserDetails() {
-  const res = await axios.post<{
-    data?: {
-      me?: {
-        uid: string;
-        displayName: string;
-        email: string;
-        photoURL: string;
-        isAdmin: boolean;
-        createdOn: string;
-        // emailVerified: boolean
-      };
-    };
-    errors?: Array<{
-      message: string;
-    }>;
-  }>(
-    `${import.meta.env.VITE_BACKEND_GQL_URL}`,
-    {
-      query: `query Me {
-      me {
-        uid
-        displayName
-        email
-        photoURL
-        isAdmin
-        createdOn
-      }
-    }`,
-    },
-    {
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      withCredentials: true,
-    }
-  );
 
-  return res.data;
-}
 
+const getInitialUserDetails = async () => {
+  const res = await authQuery.getUserDetails();
+    return res.data;
+  }
 const isGettingInitialUser: Ref<null | boolean> = ref(null);
 
-function setUser(user: HoppUser | null) {
+const setUser = (user: HoppUser | null) => {
   currentUser$.next(user);
   probableUser$.next(user);
-
   setLocalConfig('login_state', JSON.stringify(user));
-}
+};
 
-async function setInitialUser() {
+const setInitialUser = async () => {
   isGettingInitialUser.value = true;
   const res = await getInitialUserDetails();
 
-  const error = res.errors && res.errors[0];
-
-  // no cookies sent. so the user is not logged in
-  if (error && error.message === 'auth/cookies_not_found') {
-    setUser(null);
-    isGettingInitialUser.value = false;
-    return;
-  }
-
-  // cookies sent, but it is expired, we need to refresh the token
-  if (error && error.message === 'Unauthorized') {
-    const isRefreshSuccess = await refreshToken();
+  if (res.errors && res.errors[0]) {
+    const error = res.errors[0];
 
-    if (isRefreshSuccess) {
-      setInitialUser();
-    } else {
+    if (error.message === 'auth/cookies_not_found') {
```

Comment:
I can find these error messages maintained in the backend. Is it possible to refer to it from the fronted?

https://github.com/hoppscotch/hoppscotch/blob/65884293be0f1124c3db0c3fe985fe78b307d787/packages/hoppscotch-backend/src/errors.ts#L527